default/python/AI/FreeOrionAI.py

```diff
@@ -4,6 +4,9 @@
 import sys
 import random
 
+from common import configure_logging
+configure_logging.redirect_logging_to_freeorion_logger()
```

Comment:
Hm, can we move that line into `configure_logging.py`? If it gets executed immediately after import everywhere we use `configure_logging.py` anyway, it can be placed in the module to be executed on import automatically as well.